src/views/AnnualReport.vue

```diff
@@ -239,15 +239,15 @@
     >
       <div class="buttons-left">
         <v-btn id="ar-save-btn" large
-          v-if="isAnnualReportEditable"
+          v-if="isCurrentFilingEditable"
```

Comment:
I renamed this getter (since it's not specific to AR). It checks that the current filing is NEW or DRAFT.